src/app/pages/payment/payment.component.ts

```diff
@@ -0,0 +1,116 @@
+import { Component } from '@angular/core';
+import {
+  AbstractControl,
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  FormGroupDirective,
+  NgForm,
+  ValidationErrors,
+  ValidatorFn,
+  Validators,
+} from '@angular/forms';
+import { ErrorStateMatcher } from '@angular/material/core';
+import { AuthService } from '../../services/auth.service';
+
+export class MyErrorStateMatcher implements ErrorStateMatcher {
```

Comment:
Please use the ErrorStateMatcher from the authService instead using:
`import { MyErrorStateMatcher } from 'src/app/services/auth.service';`

And then:
`matcher = new MyErrorStateMatcher();`

There is an example in add-new-property.component.ts

Reply:
![image](https://github.com/amczuboka/Decagon-CondoManagementSystem/assets/98068695/96292ca0-39a3-4904-9563-cb8e17c03765)
Rectified, pushed code to the branch!